stream/stream.go

```diff
@@ -80,3 +82,15 @@ type GcpImage struct {
 	Family  string `json:"family,omitempty"`
 	Name    string `json:"name,omitempty"`
 }
+
+// ReplicatedObject represents an object in all regions of an IBMCloud-like cloud
+type ReplicatedObject struct {
+	Regions map[string]RegionObject `json:"regions,omitempty"`
+}
+
+// RegionObject represents an IBMCloud/PowerVS cloud image
+type RegionObject struct {
+	Object string `json:"object,omitempty"`
+	Bucket string `json:"bucket,omitempty"`
+	Url    string `json:"url,omitempty"`
```

Comment:
Will these ever be empty?  If not, we could drop the `omitempty` for clarity.

Reply:
yeah they won't. changed.